setup.py

```diff
@@ -29,7 +29,7 @@
 requirements = [
     "pyarrow>=10.0.1,<11.0",
     "networkx>=2.6,<3.0",
-    "openpyxl>=3.0.7,<4.0",
+    "openpyxl<3.1.0",
```

Comment:
should we have a min version as well to be sure? 

```suggestion
    "openpyxl>=3.0.7,<3.1",
```

Reply:
that certainly makes sense!